services/app-api/src/testHelpers/gqlHelpers.ts

```diff
@@ -98,13 +106,19 @@ const constructTestPostgresServer = async (opts?: {
         })
 
     await insertUserToLocalAurora(postgresStore, context.user)
+    const s3LocalClient = await newLocalS3Client(
```

Comment:
why is the default the LocalS3Client when it seems like so many tests are using the mock one instead? Are there any tests that are actually using this local client? Could the default for tests be the mock?